compiler/ras/ILValidator.cpp

```diff
@@ -39,6 +39,7 @@
 
 TR::ILValidator::ILValidator(TR::Compilation *comp)
```

Comment:
Fix the spelling in the commit title and message of `TR::ILValidator`  (`A` should be lowercase to match the class).